app/assets/javascripts/student_profile/provided_by_educator_dropdown.js

```diff
@@ -58,12 +61,13 @@
           self.props.onUserDropdownSelect(ui.item.value);
         }
       });
+
+      $('body').bind('click.closeMenu', this.closeMenu);
```

Comment:
Cool, thanks for the link!  I'm figuring you're talking about [this](https://github.com/facebook/react/issues/579#issuecomment-68906902)?  You're originally solution seems perfectly idiomatic to me, maybe you could clarify what you mean and what `__myapp_container` is?  I think you mean that is a reference to the top-level DOM node of the React app, but not quite sure.

If this helps clarify on my end, my suggestions were:
  1. Change from binding on the body to `$(React.findDOMNode(this)).bind('click', this.closeMenu)` which scopes the event, but only handles clicks within this component
  2. To make the event namespace more explicit, maybe something like `$('body').bind('click.closeProvidedByEducatorDropdownMenu', this.closeMenu);`, since with the more verbose event namespace, it's very unlikely to ever collide.

No strong feelings either way!